src/app/cli/src/graphql.ml

```diff
@@ -77,8 +77,8 @@ module Make (Program : Coda_inputs.Main_intf) = struct
                 |> User_command_memo.to_string ) ] )
 
     let snark_fee : (t, Transaction_snark_work.t option) typ =
-      obj "snark_fee" ~fields:(fun _ ->
-          [ field "snark_creator" ~typ:(non_null string)
+      obj "snarkFee" ~fields:(fun _ ->
```

Comment:
`obj` names should be capitalized in order to match our schema (you can test by doing `gq --introspect` on the daemon graphql server and see how it matches the GraphQL RFC)